src/component/App.css

```diff
@@ -0,0 +1,22 @@
+.App-container {
+  width: 100vw;
+  height: 100vh;
+  padding: 50px 100px;
+  display: grid;
+  grid-template-columns: 100%;
+  align-items: center;
+  justify-content: center;
+  background-color: #f5f6fa;
+}
+
+.page-container {
+  width: 100%;
+  height: 100%;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: #fafafa;
+}
+
+.content-container {
+  margin: 40px 80px;
+}
```

Comment:
- [ ] <b>REQUIRED:</b> Your app still doesn't look exactly like the provided design. remember, when a client gives you a project, you are expected to deliver it exactly as they requested it. kindly try to make your app's interface look more like the mockup design.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_9_6_2021_7-04-18 PM](https://user-images.githubusercontent.com/45217364/132247964-e3765e38-1f5a-4940-8696-b6cf02b7648e.png) |


  - Categories filter selector should be in the toolbar as it is the design. and it should be functional.
  - Inside the content, the margin-left and right must be the same.
  - There must be a space between the book list and a horizontal line that is above the form.
  - The profile icon must be as it is in the design.